modules/core/04-channel/keeper/events.go

```diff
@@ -436,3 +436,21 @@ func emitChannelUpgradeCancelEvent(ctx sdk.Context, portID string, channelID str
 		),
 	})
 }
+
+// emitChannelFlushCompleteEvents emits an flushing event.
+func emitChannelFlushCompleteEvents(ctx sdk.Context, portID string, channelID string, currentChannel types.Channel) {
+	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
i think it would be good to add a test for this event emission in the packet and timeout unit tests similar to [this test](https://github.com/cosmos/ibc-go/blob/fa9757635cde5bf9ca6f5a92df0c49c3b10cfc47/modules/core/04-channel/keeper/upgrade_test.go#L116), but this can also be done as part of a followup pr!

Reply:
oh i'm adding the test for the flushing complete event since it's relevant to this PR, but happy to move to other issues for other events in general as well

Reply:
i think we only have one other event to test for, so it's fine if you just cover the flushing complete event in this pr and i'll finish up the other one! thanks <3